keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -239,17 +206,17 @@ benefits of serving consistent reads from cache. For example, the kubelet LIST
 pods use case would be handled, as would similar cases. Not all cases would
 be handled.
 
-#### Option: Enable pagination in the watch cache
+#### Future work: Enable pagination in the watch cache
 
 The problem is that the watch cache ("isn't able to perform
 continuations")[https://github.com/kubernetes/kubernetes/blob/789dc873f6816cc2b9b39e77a9b94f478d3a3134/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L595].
 The watch cache is designed to only serve LIST requests at [the latest resource version is has
 available](https://github.com/kubernetes/kubernetes/blob/789dc873f6816cc2b9b39e77a9b94f478d3a3134/staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go#L115).
 
-To supporting watch cache pagination:
+To support watch cache pagination:
```

Comment:
Let's maybe remove this and just link the issue here: https://github.com/kubernetes/kubernetes/issues/108003

Reply:
Done.